api_tests/package.json

```diff
@@ -19,7 +19,7 @@
     "eslint": "^8.40.0",
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^29.5.0",
-    "lemmy-js-client": "0.18.3-rc.3",
+    "lemmy-js-client": "https://github.com/LemmyNet/lemmy-js-client#f165ef6538f6f471a923d412665d01d0e46de9b3",
```

Comment:
Fine for now I spose, but I'll have to merge that in lemmy-js-client and do another release.